packages/pwa-kit-dev/src/ssr/server/test_fixtures/node_modules/another-extension/setup-server.js

```diff
@@ -4,11 +4,25 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-/* istanbul ignore next */
-const extension = ({app}) => {
-    app.get('/another-extension', (req, res) => {
-        res.send('test')
-    })
+
+class AnotherExtension {
```

Comment:
Oh I think I see why. Is it because an app extension will be downloaded and live inside the node_modules?

I was just confused because the test fixture in the other package (pwa-kit-runtime) does not have the node_modules folder in their test fixture.

Reply:
Yeah, I think that's why. I guess it's the way Kevin figured out to get the tests working on the dev server. I'm just updating the current approach to use the class interface.